static/skin/index.js

```diff
@@ -453,6 +454,24 @@
         }
     }
 
+    function updateNavVisibilityState() {
+        const st = window.scrollY;
+        const scrollPossible = document.querySelector('.kiwixHomeBody').offsetHeight > window.screen.height;
+        const documentWidthIsLessThan590 = document.body.clientWidth < 590;
+        if ((Math.abs(previousScrollTop - st) <= 5) || !scrollPossible || !documentWidthIsLessThan590)
+            return;
+        const kiwixNav = document.querySelector('.kiwixNav');
+        if (st > previousScrollTop) {
+            kiwixNav.style.position = 'fixed';
+            kiwixNav.style.top = '-100%';
+            document.querySelector('.kiwixHomeBody').style.width = '100%';
```

Comment:
Why do you need this line here? Is the width of `kiwixHomeBody` element set dynamically to something else somewhere else?

Reply:
I don't think (and have checked) it's dynamically set anywhere. But still, when I remove `kiwixNav` all the list, gets shifted to the left on scrolling. Checking through inspector, the width is reduced hence the line.

Reply:
Can't it be set in the CSS instead?

Reply:
You're right. Thank you. At such moments, I realise I shouldn't haste.